src/pages/settings/bank-accounts/components/Details.tsx

```diff
@@ -0,0 +1,45 @@
+/**
+ * Invoice Ninja (https://invoiceninja.com).
+ *
+ * @link https://github.com/invoiceninja/invoiceninja source repository
+ *
+ * @copyright Copyright (c) 2022. Invoice Ninja LLC (https://invoiceninja.com)
+ *
+ * @license https://www.elastic.co/licensing/elastic-license
+ */
+
+import { useFormatMoney } from 'common/hooks/money/useFormatMoney';
+import { useCurrentCompany } from 'common/hooks/useCurrentCompany';
+import { BankAccDetails } from 'common/interfaces/bank-accounts';
+import { useTranslation } from 'react-i18next';
+import { Card, Element } from '../../../../components/cards';
+
+const Details = ({ accountDetails }: { accountDetails: BankAccDetails }) => {
```

Comment:
- No arrow functions.
- Export props to interface declaration, always.

```ts
interface Props {
  accountDetails: BankAccDetails
}
```

Reply:
> * No arrow functions.
> * Export props to interface declaration, always.
> 
> ```ts
> interface Props {
>   accountDetails: BankAccDetails
> }
> ```

Fixed